sql/src/main/java/org/apache/druid/sql/calcite/rule/logical/DruidAggregateCaseToFilterRule.java

```diff
@@ -133,7 +133,6 @@ public void onMatch(RelOptRuleCall call)
               .convert(aggregate.getRowType(), false);
 
     call.transformTo(relBuilder.build());
-    call.getPlanner().setImportance(aggregate, 0.0);
```

Comment:
?

Reply:
setImportance is not used anymore. This should be replaced by `call.getPlanner().prune(aggregate);`